core/common/src/main/java/alluxio/collections/LockPool.java

```diff
@@ -58,7 +59,18 @@
   private static final String EVICTOR_THREAD_NAME = "LockPool Evictor";
 
   private final Map<K, Resource> mPool;
-  private final Function<? super K, ? extends ReentrantReadWriteLock> mDefaultLoader;
+  // TODO(baoloongmao): make it configurable before merge this PR.
```

Comment:
it's fine to hard code this constant for now.
unless you feel strong there is a need to configure this parameter in practise

Reply:
OK